### DISCUSSION TITLE
ld-input

### DISCUSSION THREAD

**Discussion by ArpithaKamath** [emdgroup-liquid/liquid#386] [Q&A] [answered]
Hi,
I want to use ld-input for input of a number and was wondering why input element doesn't have any arrows for increasing/decreasing numbers? Then, what's the point of step property?
Any help is appreciated.

**Comment by borisdiakur** [comment, marked as answer]
Hello Arpitha,
the reason for why we hide the arrows is that we cannot style them consistently across different browsers and we were not happy with their default style in any of the major browsers. Implementing custom arrow buttons is a bit tricky, because they don't make much sense on touch devices, as they would be too small anyway, so you would need to detect touch devices and hide them accordingly. However, we still offer the possibility to use `type='number'` on the `ld-input` component as well as the `step` attribute in order to support input via keyboard. Please try it out using the arrow up and down buttons on your keyboard. If you think the arrow buttons are an important feature that we shouldn't take away from the user, please let us know. Your feedback is much appreciated.
